### ISSUE TITLE
Hitomi.la don't work

### ISSUE THREAD

**Issue by yami-no-tusbas** [HDoujinDownloader/HDoujin-Downloader#6]
exemple doujin : https://hitomi.la/doujinshi/fondness-doll-happy-end-english-1069167.html
Result = an empty folder fater a faw minutes, your program don't download a thing.
Here is your log : 
[1605373410-debug.log](https://github.com/HDoujinDownloader/HDoujin-Downloader/files/5541151/1605373410-debug.log)

**Comment by HDoujinDownloader** [comment]
Thanks a lot for including the log file! The error that keeps popping up is:
```csharp
System.UnauthorizedAccessException: L'accès au chemin d'accès 'X:\H\Manga\[hal]Fondness Doll Happy END (original) [English]\Fondness Doll Happy END_00.jpg' est refusé.
```
This can come up for a number of reasons, such as HDoujin Downloader not having permission to access the download directory, or the directory being invalid. Does it look right to you? It might be worth trying a different directory and seeing if that works for you.

**Comment by yami-no-tusbas** [comment]
Yes I know about permission, It was working fine before, suddently don't work anymore. I checked the perms and tested on a new folder with all perm for everyone including the app. The problem seems to be on the server side, if you go to an url like : https://bb.hitomi.la/images/c/97/729909781bfc6a3610f6642301a9f2ba261e036a539873aea2f5a9be97ec097c.png you get a 403 error. i've even tried to re-install the app. I'll can try into a virtual machine to be sure, maybe something on my windows side.

Note that the only thing I've changed since my last use of your app was setting up the use of long paths on windows 10 for another app. But should not be relevent since the pat is ~90 character and default should be 255, and more with longpath. I'll check on a VM just to be sure.

ps : I've event tried your app with admin privileges, didn't change a thing.
